app/models/plan.rb

```diff
@@ -164,8 +164,11 @@ class Plan < ActiveRecord::Base
 
   scope :search, lambda { |term|
     search_pattern = "%#{term}%"
-    joins(:template)
+    joins(:template, roles: [user: :org])
+    .where(Role.creator_condition)
```

Comment:
Not sure, but would be good to double check if this has an impact on searching for plans shared with someone who is not a creator? For example if you shared a plan with me as a commenter will this `where` prevent your plan from showing up in my search results?

Reply:
I think this would prevent any plans which were not created by the organization from showing up.  
Before, the initial scope was `current_user.org.plans` which selects along the following logic: https://github.com/DMPRoadmap/roadmap/blob/2aec36cfff386e79882584ce541d0fc46f452db4/app/models/org.rb#L241-L247

I think if this was changed to the `administrator_condition` it would have the side-effect of adding non-unique `plan` objects to the produced scope as there can be multiple `administrators` but only one `creator`.  In theory, this would be fine, but the extension is needed to query along the org-name column of the org-admin-plans-table.  Since we're displaying:
 https://github.com/DMPRoadmap/roadmap/blob/163668cea3b3545a83cde215a487ab0d3441076a/app/views/org_admin/plans/index.html.erb#L27
I think `creator_condition` might fit better here.